src/extension.ts

```diff
@@ -29,6 +30,12 @@ export async function activate(context: vscode.ExtensionContext) {
 					case "note":
 						panel.webview.html = getWebviewNote(panel.webview, context);
 						return;
+					case "deleteFolder":
```

Comment:
Den her skal nok rykkes ned til switch (message.command). Du skal nok sikre merge dev ind for at få det nyeste